python/context/ball_trajectories.py

```diff
@@ -1,92 +1,451 @@
+                    trajectory.append(position)
+            return np.array(time_stamps, np.uint), np.array(trajectory, np.float32)
+
+        def _read_folder(tennicam_path: pathlib.Path) -> StampedTrajectories:
+            """
+            List all the file in tennicam_path that have the tennicam_ prefix,
+            parse them and returns the corresponding list of stamped trajectories.
+            """
+            files = _list_files(tennicam_path, prefix="tennicam_")
+            stamped_trajectories = [_read_trajectory(tennicam_path / f) for f in files]
+            return stamped_trajectories
+
+        def _save_trajectory(
+            group: h5py._hl.group.Group,
+            index: int,
+            stamped_trajectory: StampedTrajectory,
+        ):
+            """
+            Create in the group a new subgroup named according to the index
+            and add to it 2 datasets, "time_stamps" (list of microseconds
+            time stamps) and "trajectory" (list of corresponding 3d positions)
+            """
+            # creating a new group for this trajectory
+            traj_group = group.create_group(str(index))
+            # adding 2 datasets: time_stamps and positions
+            time_stamps = stamped_trajectory[0]
+            positions = stamped_trajectory[1]
+            traj_group.create_dataset(self._TIME_STAMPS, data=time_stamps)
+            traj_group.create_dataset(self._TRAJECTORY, data=positions)
+
+        # reading all trajectories present in the directory
+        stamped_trajectories = _read_folder(tennicam_path)
+
+        # adding the new group to the hdf5 file
+        group = self._f.create_group(group_name)
+
+        # adding all trajectories as datasets to this group
+        for index, stamped_trajectory in enumerate(stamped_trajectories):
+            _save_trajectory(group, index, stamped_trajectory)
+
+    def add_json_trajectories(
+        self, group_name: str, json_path: pathlib.Path, sampling_rate_us: int
+    ) -> None:
+        """
+        It is assumed that json_path is a directory hosting (non recursively)
+        a collection of files named *.json. Each file host the (string) representation of
+        a dictionary with the key "obj" associated to an list of a 6d array
+        (3d position and 3d velocities).
+        This function will parse all these files and add them to the hdf5 under the specified
+        group name (or raise a FileNotFoundError if tennicam_path does not
+        exists). (note: the velocities values are ignored, and the time stamp list
+        is created based on the sampling rate)
+        """
+
+        def _read_trajectory(json_file: pathlib.Path) -> Trajectory:
+            """
+            Parse the json file and return the trajectory it
+            hosts.
+            """
+            with open(json_file, "r") as f:
+                content = f.read()
+            content = content.strip()
+            d = eval(content)["ob"]
```

Comment:
The docstring above mentions a key "obj", here "ob" is used.  I guess one of them is wrong?